common/config/azure-pipelines/ci.yaml

```diff
@@ -21,14 +21,14 @@ jobs:
     condition: succeeded()
     strategy:
       matrix:
-        "Windows - Node 16":
-          NodeVersion: 16.x
+        "Windows - Node 16.20.0":
+          NodeVersion: 16.20.x
```

Comment:
youre going to need to pin it down to `16.20.0`; `16.20.1` is bad

Reply:
Right. That was my first try but it didn't seem to like it... tacking another .x on the end

From failed pipeline:
"Starting: Use Node 16.20.0.x"

Reply:
need to update 
![image](https://github.com/iTwin/viewer-components-react/assets/11051042/9a50f7d5-203f-4d77-9408-e1bcdfc1cfe2)

Reply:
Thanks for the tip. Looking much better now.